samples/monster_generated.h

```diff
@@ -4,6 +4,9 @@
 #ifndef FLATBUFFERS_GENERATED_MONSTER_MYGAME_SAMPLE_H_
 #define FLATBUFFERS_GENERATED_MONSTER_MYGAME_SAMPLE_H_
 
+#pragma clang system_header
```

Comment:
why is this necessary?

Reply:
xcode treat clang extension as error, so this will stop report _Nullable or not use _Nullable as error.